openedx/features/badging/tests/test_views.py

```diff
@@ -82,11 +80,16 @@ def test_my_badges_denies_anonymous(self):
         response = Client().get(path=path)
         self.assertRedirects(response, '{}?next={}'.format(reverse('signin_user'), path))
 
+        path = reverse('trophycase')
+        response = Client().get(path=path)
```

Comment:
is there any specific reason for not using `self.client`? if not then how about to be consistent and using self.client here as well?

Reply:
Corrected it now.